File: 프로그래머스/0/120837. 개미 군단/개미 군단.java
class Solution {
    public int solution(int hp) {
        int q = hp / 5;
        int s = hp % 5;
        int answer =  hp % 5 != 0 ?
            s % 3 != 0 ?
            q + (s/3) + s % 3 : q + (s/3) : q;
        
        return answer;
    }
}

File: 프로그래머스/0/120837. 개미 군단/개미 군단.py
def solution(hp):
    answer = 0
    q = hp // 5;
    s = hp % 5;
    answer =  hp % 5 != 0 and s % 3 != 0 and q + (s//3) + s % 3 or q + (s//3) or q;
        
    
    return answer
